orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -447,3 +423,70 @@ def _create_ort_training_session(self):
         # I/O bindings
         self._train_io_binding = self._training_session.io_binding()
         self._eval_io_binding = self._training_session.io_binding()
+
+    def _init_onnx_model(self, inputs):
+        if self._onnx_model is not None:
+            return
+
+        if self._torch_model is not None:
+            # PyTorch model is moved to cpu to save GPU memory
+            self._torch_model.cpu()
+
+            # PyTorch buffers (created using 'register_buffer') shouldn't be trained
+            torch_buffers = list(dict(self._torch_model.named_buffers()).keys())
+            self.options.utils.frozen_weights.extend(torch_buffers)
+
+            # Export to ONNX
+            self._onnx_model = self._convert_torch_model_loss_fn_to_onnx(inputs)
+
+        self._init_session()
+
+    def _init_session(self):
+        if self._onnx_model is None:
+            return
+
+        # Perform internal post-processing
+        if self.options._internal_use.enable_internal_postprocess:
+            self._onnx_model = postprocess.run_postprocess(self._onnx_model)
+
+        # Perform user-specified post-processing
+        if self.options._internal_use.extra_postprocess:
+            self.options._internal_use.extra_postprocess(self._onnx_model)
+
+        # Create training session used by train_step
+        self._create_ort_training_session()
+
+    def _prepare_model_input(self, inputs_desc, lr, loss_scale, *inputs, **kwargs):
```

Comment:
I assume this and above 2 functions are just relocated. There is no change to it.

Reply:
Yes. Just sorted all methods by name

Reply:
`_update_onnx_model_initializers` is a new method and the other 3 were just relocated